packages/core/src/components/ui/Breadcrumb/Breadcrumb.tsx

```diff
@@ -42,7 +40,6 @@ const Breadcrumb = ({
           {name}
         </Link>
       )}
-      {...BreadcrumbWrapper.props}
```

Comment:
shouldn't we keep this line for the sake of spreading the section overridden props? if so we should import the correspondent wrapper in `useOverrideComponents` above.